### PR TITLE
plugin.audio.100fm v0.2.0

### PR DESCRIPTION
**Pull request by nire0510**
### Description
plugin.audio.100fm
Official Kodi add-on of 100FM Radius radio station (Israel).
The add-on offers live listening to dozens different musical genres: 80's, 90's, Rock, Pop, Oldies and many more.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Addtional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuesly monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.audio.100fm/addon.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+def index():
+    """
+    Main add-on popup
+    :return:
+    """
+    player_window = PlayerWindow()
+    api = API()
+
+    # stop player if track is not part of this addon:
+    if Player().isPlaying() and '.100fm.' not in Player().getPlayingFile():
```
Comment (phil65) [inline plugin.audio.100fm/addon.py]: why this?

File: plugin.audio.100fm/addon.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.100fm" name="100FM Radius Digital" version="0.2.0" provider-name="Nir Elbaz">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
```
Comment (phil65) [inline plugin.audio.100fm/addon.xml]: this should be 2.24.0

File: plugin.audio.100fm/addon.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>audio</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">100FM Radius Digital - Music, that's all</summary>
+    <summary lang="he">100FM רדיוס דיגיטל - מוזיקה זה הכל</summary>
+    <description lang="en">100FM radio station Kodi add-on. The add-on offers live listening to dozens different musical genres: 80's, 90's, Rock, Pop, Oldies and many more</description>
+    <description lang="he">תוסף קודי של תחנת הרדיו רדיוס FM100. התוסף מציע עשרות תחנות רדיו המנגנות מוזיקה לפי ז׳אנרים: שנות ה- 80, 90, ישראלי, להיטים ועוד</description>
+    <disclaimer lang="en">This is the official 100FM Radius Digital Kodi add-on</disclaimer>
+    <language>en</language>
```
Comment (phil65) [inline plugin.audio.100fm/addon.xml]: best would be to leave language field empty.

File: plugin.audio.100fm/changelog.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
+v0.2.0
+- Official 100FM Radius radio station add-on
```
Comment (phil65) [inline plugin.audio.100fm/changelog.txt]: What does "Official" mean here exactly? :)

File: plugin.audio.100fm/config.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        },
+        'per_row': 5,
+        'per_page': 10,
+        'margin': 60,
+        'pagination': {
+            'height': 128,
+            'width': 128
+        }
+    },
+    'viewmodes': {
```
Comment (phil65) [inline plugin.audio.100fm/config.py]: Please remove the viewmodes part.

File: plugin.audio.100fm/addon.py (hunk 1 above)
Comment (nire0510) [inline plugin.audio.100fm/addon.py]: There are some animations & info which need to be displayed only if 100fm tracks are being played

File: plugin.audio.100fm/addon.xml (hunk 2 above)
Comment (nire0510) [inline plugin.audio.100fm/addon.xml]: Can you please update for me or should I close this request and create a new one?

File: plugin.audio.100fm/changelog.txt (hunk 4 above)
Comment (nire0510) [inline plugin.audio.100fm/changelog.txt]: That means that in opposite to the previous version of this plugin, this time the station decided to adopt this plugin and offer this add-on to their listeners as additional platform

File: plugin.audio.100fm/addon.py (hunk 1 above)
Comment (phil65) [inline plugin.audio.100fm/addon.py]: yeah but why stop audio if someone opens the plugin?

**Comment by phil65** [comment]
I just tried the add-on, and this does not qualify as a plug-in. Please change the add-on type to script.

**Comment by nire0510** [comment]
@phil65 can you please explain why this does not qualify as a plug-in?
I have another plugin on the official Jarvis repo, which is quite similar, plugin.audio.calmradio, which was qualified as a plug-in.

Thanks

**Comment by phil65** [comment]
Sure. Plug-ins should make use of our VFS system. This add-on doesnt do that, instead it provides its own interface.

This is basically just a labelling issue, you only have to rename the folder and the addon id and change the addon extension point in addon.xml to
```
	<extension point="xbmc.python.script" library="default.py">
		<provides>executable</provides>
	</extension>
```
If you are not sure how to do this let me know, I will help you out then.

**Comment by nire0510** [comment]
OK, Ill do it. I'll close this pull request and open a new one.

**Pull request closed without merging**